src/plugins/intel_gpu/include/intel_gpu/primitives/ctc_greedy_decoder.hpp

```diff
@@ -29,6 +29,19 @@ struct ctc_greedy_decoder : public primitive_base<ctc_greedy_decoder> {
         , ctc_merge_repeated(ctc_merge_repeated)
         , output_tensor(output_tensor) {}
 
+    /// @brief Constructs ctc_greedy_decoder primitive.
+    /// @param id This primitive id.
+    /// @param input Input primitive id (input, sequence_indicators, second_output(optional)).
+    /// @param ctc_merge_repeated Flag for merging repeated labels during the CTC calculation
```

Comment:
The comment is same with existing constructor. Please specify explanation of new constructor.

Reply:
It is updated. Thanks for your comment.